crates/examples/infra/mod.rs

```diff
@@ -1038,6 +1044,7 @@ pub async fn main_entry_point<
             my_own_validator_config,
             args.advertise_address,
             Some(libp2p_public_key),
+            true,
```

Comment:
Is it possible to get a comment on the true and the false above so a reader knows what it's doing? Perhaps something like `true /* is_da */` or a constant `IS_DA`

Reply:
Yes, good point. This is actually `indexed_da` which controls whether or not being DA is governed by the node index. It's because we both get the node index and send our DA status within this function, so it needed to be done in here. It probably makes more sense to break the function out, but I wanted to make it work with minimal changes

Reply:
Just want to check my understanding, is the flow in examples be like: a node post and register its key pairs to the orchestrator -> the orchestrator give it a `node_index` and use this `node_index` to determine whether it is a da node. 
While in other scenarios it's possible that they do not use `node_index` to determine whether a node is part of DA committee, then they need to call `get_complete_config(..., false)` .

Reply:
That's correct, the only difference being that the orchestrator does not determine who is DA, that happens locally after the node gets its `node_index`. Then in the second call to the orchestrator it tells the orchestrator it is DA